dali/test/python/test_external_source_parallel_custom_serialization.py

```diff
@@ -414,20 +427,20 @@ def _test_serialization_of_globals_referenced_in_list_comprehension_in_cb(name,
 @register_case(tests_dali_pickling)
 @register_case(tests_cloudpickle_pickling)
 def _test_mutually_recursive_functions(name, py_callback_pickler):
-    div_by_2 = lambda n, acc=0 : acc if n <= 0 else add_one(n // 2, acc)
-    add_one = lambda n, acc : div_by_2(n, acc + 1)
+    def div_by_2(n, acc=0): return acc if n <= 0 else add_one(n // 2, acc)
+    def add_one(n, acc): return div_by_2(n, acc + 1)
```

Comment:
Initially, I've put `noqa` here. But since lambdas in Python (in general also) are rather meant to be anonymous, and @jantonguirao proposed this quite nice one-liner solution, I've changed it to this. I believe we can keep it this way

Reply:
They are pickled and lambdas are pickled differently. I insist, this is not just style here.

Reply:
Done